src/pages/docs/creators/javascript/examples/switching-to-mainnet.md

```diff
@@ -10,7 +10,7 @@ type: Example
 
 This section is aimed mainly at **requestors** wishing to switch from running simple test tasks on our `testnet` to launching production payloads utilizing the vast number of providers on the `mainnet`.
 
-By default, the JS SDK will execute your task on the `testnet`. It is a development network with a limited amount of providers and limited amounts of computational resources available. If you would rather need a larger amount of providers to compute on, you should switch to the mainnet.
+By default, the JS SDK will execute tasks on the `testnet`. It is a development network with a limited amount of providers and limited amounts of computational resources available. If you would rather need a larger amount of providers to compute on, you should switch to the mainnet.
```

Comment:
This is the new text we chose to update on a call together some time ago. This was replacing the other one where i sent the diff link